src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/Regex.cs

```diff
@@ -77,7 +77,7 @@ internal Regex(string pattern, RegexOptions options, TimeSpan matchTimeout, Cult
             if ((options & RegexOptions.NonBacktracking) != 0)
             {
                 // If we're in non-backtracking mode, create the appropriate factory.
-                factory = new SymbolicRegexRunnerFactory(_code, options, matchTimeout, culture);
+                factory = new SymbolicRegexRunnerFactory(_code, options, matchTimeout, culture, capsize, caps);
```

Comment:
Why do you need the extra two params? Aren't these two part of the already passed in _code? Or is the idea here to not use _code any more?

Reply:
They indeed are available in _code, thanks!

Reply:
They do seem like they are just set from _code directly, and tests do pass.

Reply:
Ok, cool.  I think I was thinking about RegexTree._caps not necessarily being the same as RegexCode._caps.